src/test/java/Entities/PlayerTest.java

```diff
@@ -50,7 +48,7 @@ public void testResume(){
     @Test
     public void testPlayAfterPause(){
         File file = new File("./src/test/java/Entities/test.mp3");
-        Song song = new Song(0, null, null, null, file, null, null);
+        Song song = new Song(0, null, null,null, file, null, null);
```

Comment:
Was this accidental? I'd just revert it so this file doesn't come up in files changed.

Reply:
oh true mb